draft-ietf-masque-connect-udp-listen.md

```diff
@@ -213,8 +215,12 @@ COMPRESSION_CLOSE with the context ID (see {{capsulecloseformat}}) if it
 doesn't wish to support compression for the given Context ID (For example, due
 to the memory cost of establishing a list of mappings per target per client).
 If the compression was rejected, the client and proxy will instead use an
-uncompressed context ID (See {{uncompressed}}) to exhange UDP payloads for the
-given target, if those have been enabled.
+uncompressed context ID (See {{uncompressed}}) to exchange UDP payloads for the
+given target, if those have been enabled. Only one Context ID can be used per
+IP-port tuple. If an endpoint detects that both itself and its peer have opened a context ID for the same tuple, the endpoint MUST close the context ID that was opened by the server. If
+a peer attempts to allocate another Context ID for a tuple which already has
+an active context ID it previously requested, this COMPRESSION_ASSIGN capsule
+MUST be considered malformed.
```

Comment:
Similar to my previous comment, the sentence is phrased in terms of what the peer does, but it's better to phrase it in terms of what the receiver knows/does.

"If an endpoint receives a COMPRESSION_ASSIGN capsule whose tuple matches another open context ID, it MUST treat the capsule as malformed.